### DISCUSSION TITLE
Does cache has been removed in the latest release?

### DISCUSSION THREAD

**Discussion by AhmedAbouelkher** [minio/minio#16130] [Q&A] [answered]
I've installed the latest release [RELEASE.2022-11-17T23-20-09Z](https://github.com/minio/minio/commits/RELEASE.2022-11-17T23-20-09Z) And noticed that configurations > cache had been removed, is this so?

I need an updating guide also to upgrade my existing minio version.

**Comment by harshavardhana** [comment, marked as answer]
Server version does not support caching anymore, caching will come back in future when we implement edge caching

**Reply by AhmedAbouelkher** [reply]
Ok,
What is the best practice to upgrade minIO on an active server?

**Reply by harshavardhana** [reply]
https://min.io/docs/minio/linux/operations/install-deploy-manage/upgrade-minio-deployment.html
